doc/source/serve/multi-app.md

```diff
@@ -106,50 +116,53 @@ Check the status of the applications by running `serve status`.
 ```console
 $ serve status
 proxies:
-  59e6321aa1699fa09ed1094e1f0b3c37afd47fc208802db3c07328d0: HEALTHY
+  2e02a03ad64b3f3810b0dd6c3265c8a00ac36c13b2b0937cbf1ef153: HEALTHY
 applications:
-  app1:
+  classify:
```

Comment:
I think the config above is still using "app1" and "app2"?

Reply:
Yeah this output is wrong, I've changed it to app1 and app2